src/main/resources/assets/admin/common/icons/icons.less

```diff
@@ -71,10 +71,14 @@
   }
 }
 
-.@{_COMMON_PREFIX}icon-page {
+.icon-page {
   .icon-file-text2;
 }
 
+.@{_COMMON_PREFIX}icon-page {
+  .icon-page;
+}
+
```

Comment:
Instead of duplicating styles, use common prefix on the icon class:

`StyleHelper.getCommonIconCls('page'`) will give you `icon-page` with common prefix. 

https://github.com/enonic/app-contentstudio/blob/master/modules/lib/src/main/resources/assets/js/page-editor/ComponentDescriptorsComboBox.ts#L3
https://github.com/enonic/app-contentstudio/blob/master/modules/lib/src/main/resources/assets/js/page-editor/ComponentDescriptorsComboBox.ts#L84